cmd/ccboc/main.go

```diff
@@ -121,6 +121,88 @@ func getCalculationID(c *cli.Context) error {
 	return nil
 }
 
+func getCalculationResult(c *cli.Context) error {
+	teff := c.Float64("teff")
+	logG := c.Float64("logG")
+	path := c.String("path")
+
+	resp, err := http.Get(globalConfig.APIURL + "/calculations/results?teff=" + fmt.Sprintf("%0.1f", teff) + "&logg=" + fmt.Sprintf("%0.1f", logG))
+	if err != nil {
+		logrus.WithError(err).Fatal("Couldn't retrieve the data from headers.")
+	}
+
+	key := resp.Header.Get("filename")
+
+	var defaultPath string
+
+	if path != "" {
+		defaultPath = filepath.Join(path, filepath.Base(key))
+	} else {
+		defaultPath = filepath.Join("~/.config/ccboc/", filepath.Base("text.txt"))
```

Comment:
We should try to get the `$HOME` path like
```go
	user, err := user.Current()
	if err != nil {
		log.Fatalf(err.Error())
	}
	home := user.HomeDir 
``` 

Also, `filepath.Base("text.txt")` will always return `text.txt` but it is also wrong since the file will never be just a text file.